deebot_client/commands/json/pads_cleaning_interval.py

```diff
@@ -0,0 +1,39 @@
+"""Pads cleaning interval commands."""
+from typing import Any
+
+from deebot_client.command import InitParam
+from deebot_client.event_bus import EventBus
+from deebot_client.message import HandlingResult, MessageBodyDataDict
+
+from .common import CommandWithMessageHandling, SetCommand
+
+
+class GetPadsCleaningInterval(CommandWithMessageHandling, MessageBodyDataDict):
+    """Get pads cleaning interval command."""
+
+    name = "getPadsCleaningInterval"
+
+    @classmethod
+    def _handle_body_data_dict(
+        cls, event_bus: EventBus, data: dict[str, Any]
+    ) -> HandlingResult:
+        """Handle message->body->data and notify the correct event subscribers.
+        :return: A message response
+        """
+        event_bus.notify(
+            PadsCleaningIntervalEvent(PadsCleaningInterval(int(data["interval"])))
+        )
+        return HandlingResult.success()
+
+
+class SetPadsCleaningInterval(SetCommand):
+    """Set pads cleaning interval command."""
+
+    name = "setPadsCleaningInterval"
+    get_command = GetPadsCleaningInterval
+    _mqtt_params = {"interval": InitParam(PadsCleaningInterval)}
+
+    def __init__(self, interval: PadsCleaningInterval | str) -> None:
+        if isinstance(interval, str):
+            interval = PadsCleaningInterval.get(interval)
+        super().__init__({"interval": interval.value})
```

Comment:
Can you set the interval to any minute? E.g. to 7 minutes?

If yes, I would suggest you to refactor the the command like the volume commands, where the user can specify any number

Reply:
Yes you can set any amount of minutes. I did want to develop the cleaning pads interval command to match the volume command however I couldn't because I used the code you merged on PR #449 for work mode and copied what you did without understanding it. I don't have such "delta" with the volume command so I am lost on so many levels...